src/test/java/stepDefinitions/StepsArticles.java

```diff
@@ -87,4 +96,69 @@ public void iClickTheDeleteButton() {
     public void theArticleWasSuccessfullyRemoved() {
         Assert.assertEquals(ARTICLE_DELETE_ERROR, ARTICLES_DELETE_MSG, articleReadPage.getAlertMsg());
     }
+
+    @And("the system displays the list of items")
+    public void theSystemDisplaysTheListOfItems() {
+        Assert.assertTrue(articlesFeedPage.articlesTitles().size() > 0);
```

Comment:
You can use the method `isEmpty` to avoid comparing if the size is greater than zero.

Reply:
it is changed  `Assert.assertFalse(articlesFeedPage.articlesTitles().isEmpty());`